### PR TITLE
Cleanup/refactor node code

### PR DESCRIPTION
**Pull request by bitwiseguy**
A few areas of cleanup:

1. Make the `Sent message` log after the message is successfully sent
2. Use `messageOpts` and `storeOpts` when initializing those services. This means we pass a single struct as an input to `NewStore` and `NewMessageService` instead of many individual arguments. This work aligns `messageservice` and `store` factories with the `chainservice.NewEthChainService` factory

### CODE REVIEW

File: node_test/rpc_test.go [hunk 1]
```diff
@@ -99,6 +98,7 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 	for i := 0; i < n; i++ {
 		sk := `000000000000000000000000000000000000000000000000000000000000000` + strconv.Itoa(i+1)
 		actors[i] = ta.Actor{
+			// PrivateKey: common.Hex2Bytes(sk),
```
Comment (geoknee) [inline node_test/rpc_test.go:101]: ✂️

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,12 @@
 docker/cloud/build:
 	docker build -f docker/cloud/Dockerfile -t go-nitro-cloud .
+
 docker/cloud/start:
 	docker remove go-nitro-cloud || true
-	docker run -it -d --name go-nitro-cloud -p 3005:3005 -p 4005:4005 -p 5005:5005 -e NITRO_CONFIG_PATH="./nitro_config/iris.toml" go-nitro-cloud
+	docker run -it -d --name go-nitro-cloud \
+	  -p 3005:3005 -p 4005:4005 -p 5005:5005 \
+		-e NITRO_CONFIG_PATH="./nitro_config/charley.toml" \
```
Comment (geoknee) [inline Makefile:8]: Did you mean to change this from `iris` to `charley`?

**Review by geoknee — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (bitwiseguy) [reply Makefile]: It was changed to `charley` for some local testing. Changed back in https://github.com/statechannels/go-nitro/pull/1747/commits/f9a9b81b920d412edb92208ad0735c8b674bd0d9

File: node_test/rpc_test.go (hunk 1 above)
Comment (bitwiseguy) [reply node_test/rpc_test.go]: Fixed in https://github.com/statechannels/go-nitro/pull/1747/commits/f9a9b81b920d412edb92208ad0735c8b674bd0d9

**Pull request merged**